polyfile/http/http_11.py

```diff
@@ -1,11 +1,20 @@
-from enum import StrEnum
 import importlib
 from string import whitespace
+import sys
 
 from abnf.grammars.misc import load_grammar_rules
 
 from ..logger import getStatusLogger
 
+if sys.version_info < (3, 11):
```

Comment:
oh shoot sorry about this, and thanks for accommodating it - definitely an artifact of not entirely finishing